index.html

```diff
@@ -1887,324 +1887,639 @@ <h2>
         </dd>
       </dl>
     </section>
-    <section data-dfn-for="PaymentAddress" data-link-for="PaymentAddress">
+    <section>
       <h2>
-        <dfn>PaymentAddress</dfn> interface
+        Physical Addresses
       </h2>
-      <pre class="idl">
-        [SecureContext, Exposed=Window]
-        interface PaymentAddress {
-          [Default] object toJSON();
-          readonly attribute DOMString country;
-          readonly attribute FrozenArray&lt;DOMString&gt; addressLine;
-          readonly attribute DOMString region;
-          readonly attribute DOMString city;
-          readonly attribute DOMString dependentLocality;
-          readonly attribute DOMString postalCode;
-          readonly attribute DOMString sortingCode;
-          readonly attribute DOMString languageCode;
-          readonly attribute DOMString organization;
-          readonly attribute DOMString recipient;
-          readonly attribute DOMString phone;
-        };
-      </pre>
-      <section>
+      <p>
+        A <dfn>physical address</dfn> is composed of the following concepts.
+      </p>
+      <dl data-sort="">
+        <dt>
+          <dfn>Country</dfn>
+        </dt>
+        <dd>
+          The country corresponding to this address.
+        </dd>
+        <dt>
+          <dfn>Address line</dfn>
+        </dt>
+        <dd>
+          The most specific part of the address. It can include, for example, a
+          street name, a house number, apartment number, a rural delivery
+          route, descriptive instructions, or a post office box number.
+        </dd>
+        <dt>
+          <dfn>Region</dfn>
+        </dt>
+        <dd>
+          The top level administrative subdivision of the country. For example,
+          this can be a state, a province, an oblast, or a prefecture.
+        </dd>
+        <dt>
+          <dfn>City</dfn>
+        </dt>
+        <dd>
+          The city/town portion of the address.
+        </dd>
+        <dt>
+          <dfn>Dependent locality</dfn>
+        </dt>
+        <dd>
+          The dependent locality or sublocality within a city. For example,
+          used for neighborhoods, boroughs, districts, or UK dependent
+          localities.
+        </dd>
+        <dt>
+          <dfn>Postal code</dfn>
+        </dt>
+        <dd>
+          The postal code or ZIP code, also known as PIN code in India.
+        </dd>
+        <dt>
+          <dfn>Sorting code</dfn>
+        </dt>
+        <dd>
+          The sorting code as used in, for example, France.
+        </dd>
+        <dt>
+          <dfn>Language code</dfn>
+        </dt>
+        <dd>
+          The language in which the address is provided. It's used to determine
+          the field separators and the order of fields when formatting the
+          address for display.
+        </dd>
+        <dt>
+          <dfn>organization</dfn>
```

Comment:
The capitalization is inconsistent on these components of an address. I think consistently lowercase would be best.